content/App.tsx

```diff
@@ -32,23 +33,23 @@ const queryClient = new QueryClient({
 });
 
 const InnerApp: FC<AppProps> = ({ Component, pageProps }) => {
-	const { provider, signer, network, L1DefaultProvider } = Connector.useContainer();
+	const { provider, signer, network, L1DefaultProvider, synthetixjs } = Connector.useContainer();
 
 	useEffect(() => {
 		try {
 			document.querySelector('#global-loader')?.remove();
 		} catch (_e) {}
 	}, []);
-
+	const networkId = String(network?.id);
 	return (
 		<>
 			<SynthetixQueryContextProvider
 				value={
-					provider && network?.id
+					provider && isSupportedNetworkId(networkId) && synthetixjs
 						? createQueryContext({
 								provider: provider,
 								signer: signer || undefined,
-								networkId: network.id,
+								networkId,
 						  })
 						: createQueryContext({
 								networkId: 1,
```

Comment:
This will run when network id is not supported - do we want that to occur? Shouldn't no queries run if we are not on a supported network

Reply:
Not the clearest code. But since the tenary is this:`provider && isSupportedNetworkId(networkId) && synthetixjs` 
It means we will be on a supported network since otherwise synthetixjs would be `undefined`